crypto/ledger/ledger_test.go

```diff
@@ -73,10 +73,10 @@ func TestPublicKeyUnsafeHDPath(t *testing.T) {
 		require.NoError(t, tmp.ValidateKey())
 		(&tmp).AssertIsPrivKeyInner()
 
-		pubKeyAddr, err := legacybech32.MarshalPubKey(legacybech32.AccPK, priv.PubKey())
+		pkBz, err := codec.ProtoMarshalJSON(priv.PubKey(), nil)
```

Comment:
we should pass an interfaceRegistry here, so that the JSONs have a `"@type"` field too

Reply:
^ agreed, otherwise we have no way to go back to the proto.Message type of the key

Reply:
For that we would need to wrap it into `Any`. the `interfaceRegistry` doesn't change anything.
But we don't need that in this test. The tests checks that the paths are resolved correctly and data is saved correctly.

Reply:
> agreed, otherwise we have no way to go back to the proto.Message type of the key

ditto, we don't test that. This tests only check path and store.  Serialization is tested in other places, eg: `multisig/multisig_test.go` and `keyring/output_test.go

Reply:
OK. Can you add a comment about this? Seeing just `{"key":"..."}` is not usual in the SDK.

Reply:
Well, I can wrap it into Any, I'm just not convinced if it makes sense to add additional code for that if this is not a goal of this test.
I will add a comment, if anyone is against, then lmk.

Reply:
I can also use `String` method, which will displays more friendly output. WDYT?

Reply:
+1 for String, we don't even need ProtoMarshalJSON.